src/industry_gui.cpp

```diff
@@ -367,16 +367,16 @@ class BuildIndustryWindow : public Window {
 	 * @param prefixstr    String to use for the first item
 	 * @return A formatted raw string
 	 */
-	std::string MakeCargoListString(const CargoID *cargolist, const CargoSuffix *cargo_suffix, int cargolistlen, StringID prefixstr) const
+	std::string MakeCargoListString(const std::span<const CargoID> cargolist, const CargoSuffix *cargo_suffix, StringID prefixstr) const
```

Comment:
cargo_suffix can also be a span, and it should be the same length as cargolist, IIRC.